### PR TITLE
Multi node combination LCBC

### PR DESCRIPTION
**Pull request by vlukes**
I'm still wondering what example to use.

### CODE REVIEW

**Comment by rc** [comment]
Thanks! The example looks good. Do you plan to add a test to `test_lcbcs.py`?

**Comment by rc** [comment]
> > Thanks! The example looks good. Do you plan to add a test to `test_lcbcs.py`?
> 
> Could it be done in [this](https://github.com/sfepy/sfepy/pull/1085/commits/3ffdd44492dec09660f08c5dd9511f115021ba59) way?

This would be the same as adding the example to `sfepy/tests/test_declarative_examples.py`, right? I was thinking more about using `field.get_dofs_in_region()` along the hanging node edge sides to verify the continuity.

File: sfepy/discrete/fem/lcbc_operators.py [hunk 1]
```diff
@@ -528,6 +526,78 @@ def __init__(self, name, regions, dof_names, dof_map_fun,
                                          ts, functions=functions)
 
 
+class MultiNodeLCOperator(LCBCOperator):
+    r"""
+    Transformation matrix operator that define the DOFs in one (depenedent) node
+    as a linear combination of the DOFs in some other (independent) nodes.
+
```
Comment (rc) [inline sfepy/discrete/fem/lcbc_operators.py:533]: Transformation matrix operator that defines the DOFs in one (dependent) node
    as a linear combination of the DOFs in some other (independent) nodes.

File: sfepy/discrete/fem/lcbc_operators.py [hunk 2]
```diff
@@ -528,6 +526,78 @@ def __init__(self, name, regions, dof_names, dof_map_fun,
...
+    The linear combination is given by:
+
+    .. math::
+        \bar u_i \sum_{j=1}^n c^{j} u_i^j\;,
+
```
Comment (rc) [inline sfepy/discrete/fem/lcbc_operators.py:538]: \bar u_i = ...

**Comment by rc** [comment]
I found some typos in the docstring, otherwise looks fine, thanks!

**Comment by vlukes** [comment]
> I found some typos in the docstring, otherwise looks fine, thanks!

Fixed.

**Pull request merged**